website/routes.py

```diff
@@ -1170,6 +1170,12 @@ def make_url_map(app):
             addon_views.addon_view_or_download_file,
             OsfWebRenderer('project/view_file.mako', trust=False)
         ),
+        Rule(
+            '/<pid>/download/<fid_or_guid>/',
```

Comment:
@laurenrevere Why is the `<pid`> necessary?

Reply:
It's used for permissions `@must_be_contributor_or_public` uses it.

Reply:
discussed in person: Remove `/pid/` from the path because it's extra and these URLs will be exposed to users.